xbmc/pvr/windows/GUIViewStatePVR.cpp

```diff
@@ -50,8 +51,8 @@ CGUIViewStateWindowPVRRecordings::CGUIViewStateWindowPVRRecordings(const int win
   if (CServiceBroker::GetPVRManager().Clients()->AnyClientSupportingRecordingsSize())
     AddSortMethod(SortBySize,  553, LABEL_MASKS("%L", "%I", "%L", "%I")); // "Size"   : Filename, DateTime | Foldername, empty
 
-  // Default sorting
-  SetSortMethod(SortByDate);
+  SetSortMethod(CServiceBroker::GetSettingsComponent()->GetAdvancedSettings()
+                ->m_PVRDefaultViewState.m_sortDescription);
```

Comment:
As you are only using the 'CViewStet::m_sortDescription' member I see no need to store a 'CViewState' in advancedsettings.xml. This shoul be a 'SortDescription'.